runt.toml

```diff
@@ -225,6 +226,7 @@ name = "memory by reference tutorial"
 paths = ["examples/futil/memory-by-reference/*.futil"]
 cmd = """
 fud e {} -s verilog.cycle_limit 500 \
+         -s futil.flags ' -d top-down-st' \
```

Comment:
This *should* retain the flag. No action needed by leaving the comment here so that we remember.